src/plugins/__init__.py

```
@@ -0,0 +1,48 @@
+from typing import Dict, Type
+from ..domain.interfaces import DataSource
+from importlib import import_module
+import logging
+
+
+logger = logging.getLogger(__name__)
+
+
+class PluginRegistry:
+    def __init__(self):
```

Ruff (0.12.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)